### DISCUSSION TITLE
Workaround for Models with Vertex colors while using AR Quicklook

### DISCUSSION THREAD

**Discussion by v-prgmr** [google/model-viewer#1924] [Q&A]
### Description

My current pipeline generates 3D models as .glb with only vertex colours instead of having a texture image file. I need to convert the model to .usdz so that I can use it in AR Quicklook. But the problem right now is that AR Quicklook doesn't support Vertex colours and my model's mesh has no vertex colours when deployed in AR using AR Quicklook. 

Can anyone suggest a workaround or a better way to do this? I am not using a separate texture file associated with the model because of size constraints. 

#### Live Demo
![WhatsApp Image 2020-10-01 at 10 58 37 PM](https://user-images.githubusercontent.com/39503553/94998794-c7d7e880-05b4-11eb-8aa8-a629443e9d9f.jpeg)

### Version

- model-viewer: v1.2.1

### Browser Affected

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Apple's reality converter is the gold standard for converting glb to usdz, so if that's not doing it, then Apple probably doesn't support it. In any case, the reason we now use textures instead of vertex colors is because they in fact result in significantly smaller file sizes for a given quality. I would highly recommend that you try textures and look into jpg encoding and reducing resolution and decimating/DRACO-encoding your mesh to get the size down.

**Comment by v-prgmr** [comment]
@elalish , thank you for your insight into this. I will look into textures and decimation/ DRACO-encoding.
